src/main/kotlin/commands/BanCommand.kt

```diff
@@ -0,0 +1,34 @@
+package commands
+
+import Command
+import ConfigManager.readConfigSafe
+import UserConfig
+import long
+import integer
+import string
+import doesLater
+import arg
+
+object BanCommand : Command("ec") {
+    init {
+        long("user") {
+            integer("deletemessagedays") {
+                string("reason") {
+                    doesLater { context ->
+                        val id: Long = context arg "user"
+                        val messageDays: Int = context arg "deletemessagedays"
+                        val reason: String = context arg "reason"
+                        val user = Main.client!!.getUser(id)
+                        val server = readConfigSafe<UserConfig>(
```

Comment:
there's no need to assign server when you can already access it from the command context, with `server`

Reply:
its still nullable